integrated_channels/blackboard/exporters/content_metadata.py

```diff
@@ -12,17 +12,13 @@
 class BlackboardContentMetadataExporter(ContentMetadataExporter):
     """
         Blackboard implementation of ContentMetadataExporter.
-        The odd choice of making enrollment_url into description with a link is best explained as:
-            Blackboard does not show description by default on home page.
-            However, description shows on the course search page.
-            This seems to be the only place to visibly place a link for now for learners to
-            find and visit the edX course page. But this can change as we explore blackboard more.
+        Note: courseId is not being exported here (instead done in client), because it will be
```

Comment:
perhaps an explanation as to why the courseId is random/why it's ok that the course id is random?

Reply:
yup I think the hash function works as designed, seeds at python subsystem startup time so we can't use it as is if we want consistent hash values ...but I can see this comment being a bit confusing here (why even mention that issue here, since there is no code here to generate courseId)